Test/gptmdPrunedBdTests.cs

```diff
@@ -50,7 +50,7 @@ public static void TestPrunedGeneration()
             //ensures that protein out put contins the correct number of proteins to match the folowing conditions. 
                 // all proteins in DB have baseSequence!=null (not ambiguous)
                 // all proteins that belong to a protein group are written to DB
-            Assert.AreEqual(20, proteins.Count);
+            Assert.AreEqual(18, proteins.Count);
```

Comment:
Why did this change?

Reply:
The original unit test used 4 different modification types (common biological, less common, common artifact and metal). The default is now changed to common biological and artifact). So there are fewer mods in the pruneddb and fewer proteins.

Reply:
Makes sense :)

Reply:
comes from the "Less Common" class of mods. Even though they are not written. This is a strangely written unit test. Writes gptmd prunded db but the db has no mods.....